README.md

```diff
@@ -68,3 +83,8 @@ import { EntityOpsLevelMaturityContent } from 'backstage-plugin-opslevel-maturit
 ## Next Steps
 
 Visit the Maturity tab in Backstage to get started.
+
+
+### Maturity of this Plugin
+
+[![Overall](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fapp.opslevel.com%2Fapi%2Fservice_level%2FSX_5tBBV3PXtQcTEe4j6kGw_Sm0ys-piO0swtoWKCwo)](https://app.opslevel.com/services/backstage_plugin/maturity-report)
```

Comment:
To pass [our check](https://app.opslevel.com/services/backstage_plugin/maturity-report?checkId=Z2lkOi8vb3BzbGV2ZWwvQ2hlY2tzOjpSZXBvRmlsZS82ODU2). Platinum here we come!